packages/shift-react-components/src/components/checkout/address-form.js

```diff
@@ -131,14 +125,14 @@ export class AddressForm extends Component {
     const fieldOption = { className: 'o-form__input-block', label: 'Company Name', name: 'companyName', value: formInput.companyName }
 
     return (
-      <>
+      <Fragment>
```

Comment:
I've noticed you change these - why not use the shorthand?

Reply:
@marcgear ahh, couple of reasons:
1. I generally try to avoid shorthand things like this, because the full version is more readable in my opinion. Probably comes from avoiding shorthand in CSS, so instead of `background: #f00;` I'd always do `background-color: #f00;`. Slightly opinionated. With JS, it _should_ all compile down to the same though
2. Sublime doesn't recognise the syntax, or the highlighter I use currently doesn't, so the rest of the file breaks (last time I checked there was only 1 syntax highlighter that worked for it, and it ended up breaking loads of other things https://github.com/borela/naomi)